activesupport/lib/active_support/core_ext/numeric/conversions.rb

```diff
@@ -99,31 +99,30 @@ module ActiveSupport::NumericWithFormat
   #  1234567.to_s(:human, precision: 1,
   #                   separator: ',',
   #                   significant: false)                   # => "1,2 Million"
-  def to_s(*args)
-    format, options = args
-    options ||= {}
-
+  def to_s(format = nil, options = nil)
     case format
+    when nil
+      super()
+    when Integer, String
+      super(format)
     when :phone
-      return ActiveSupport::NumberHelper.number_to_phone(self, options)
+      return ActiveSupport::NumberHelper.number_to_phone(self, options || {})
     when :currency
-      return ActiveSupport::NumberHelper.number_to_currency(self, options)
+      return ActiveSupport::NumberHelper.number_to_currency(self, options || {})
     when :percentage
-      return ActiveSupport::NumberHelper.number_to_percentage(self, options)
+      return ActiveSupport::NumberHelper.number_to_percentage(self, options || {})
     when :delimited
-      return ActiveSupport::NumberHelper.number_to_delimited(self, options)
+      return ActiveSupport::NumberHelper.number_to_delimited(self, options || {})
     when :rounded
-      return ActiveSupport::NumberHelper.number_to_rounded(self, options)
+      return ActiveSupport::NumberHelper.number_to_rounded(self, options || {})
     when :human
-      return ActiveSupport::NumberHelper.number_to_human(self, options)
+      return ActiveSupport::NumberHelper.number_to_human(self, options || {})
     when :human_size
-      return ActiveSupport::NumberHelper.number_to_human_size(self, options)
+      return ActiveSupport::NumberHelper.number_to_human_size(self, options || {})
+    when Symbol
+      super()
```

Comment:
I think this clause is still wrong - we should either delete it and it'll raise the appropriate error (`TypeError` from `Integer` or `ArgumentError` from `Float`), or we should raise an `ArgumentError` with an unknown format message rather than just silently returning the default. That would be a better developer experience since it'd blow up if there was a typo in the format name.